scripts/deploy-multus-network.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source $SCRIPT_DIR/init-env.sh
+
+if $IS_MINIKUBE
```

Comment:
Can we reverse the name and reflect the actual check better? It's actually "NON_OCP". Also, prefixing it with TNF_?

Reply:
fixed